src/_cffi_src/openssl/x509v3.py

```diff
@@ -34,6 +34,7 @@
 typedef ... Cryptography_STACK_OF_ASN1_INTEGER;
 typedef ... Cryptography_STACK_OF_GENERAL_SUBTREE;
 typedef ... EXTENDED_KEY_USAGE;
+typedef ... CONF;
```

Comment:
I guess it's fine here.

On Wed, Mar 9, 2016 at 7:48 AM, Paul Kehrer <notifications@github.com>
wrote:

> In src/_cffi_src/openssl/x509v3.py
> <https://github.com/pyca/cryptography/pull/2781#discussion_r55512850>:
>
> > @@ -34,6 +34,7 @@
> >  typedef ... Cryptography_STACK_OF_ASN1_INTEGER;
> >  typedef ... Cryptography_STACK_OF_GENERAL_SUBTREE;
> >  typedef ... EXTENDED_KEY_USAGE;
> > +typedef ... CONF;
>
> Not, that's just the only place we use it. We could keep conf.py and only
> have the opaque typedef in it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2781/files#r55512850>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084